Dockerfile

```diff
@@ -17,11 +17,21 @@ MAINTAINER devops@edx.org
 
 # libmysqlclient-dev; to install header files needed to use native C implementation for 
 # MySQL-python for performance gains.
-
+# software-properties-common; to get apt-add-repository
 # If you add a package here please include a comment above describing what it is used for
-RUN apt-get update && apt-get upgrade -qy && apt-get install language-pack-en locales git python3.5 python3-pip libmysqlclient-dev libssl-dev python3-dev -qy && \
-pip3 install --upgrade pip setuptools && \
-rm -rf /var/lib/apt/lists/*
+
+RUN apt-get update &&\
+    apt-get install -y locales &&\
+    apt-get install -y locales software-properties-common &&\
+    locale-gen en_US.UTF-8
+
+# Add the deadsnakes PPA to install Python 3.8
+RUN apt-add-repository -y ppa:deadsnakes/ppa
```

Comment:
Could we move this up above the first apt-get?

Reply:
Need to install software-properties-common first before using ```apt-add-repository````